### DISCUSSION TITLE
fractional land use

### DISCUSSION THREAD

**Discussion by bbakernoaa** [corteva/rioxarray#272] [Q&A]
I'm trying to figure out how to calculate the fractional land use cover using land cover dataset (lets just say this one: 

[ftp://ftp.star.nesdis.noaa.gov/pub/smcd/JPSS/VIIRS-AST/VIIRSSurfaceType4EMC_2020Nov/VIIRS_GST_2012to2019v3L2_20types_30arcsec.nc.gz ](url)

(note it isn't actually zipped its a mistake) 

I want to calculate fractional area used by each land use type in each cell.  Lets say that the projection is 

'+proj=lcc +lat_0=38.5 +lon_0=-97.5 +lat_1=38.5 +lat_2=38.5 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs +type=crs'

**Comment by snowman2** [comment]
I would recommend updating your question following the guide given for `gis.stackexchange.com`:
> When seeking help to debug/write/improve code always provide the desired behavior, a specific problem/error and the [shortest code](https://stackoverflow.com/help/mcve) (as [formatted text](https://meta.stackexchange.com/questions/22186), not pictures) needed to reproduce it in the question body. Providing a clear problem statement and a code attempt helps others to help you.
